docs/source/tutorial/index.md

```diff
@@ -0,0 +1,22 @@
+These tutorials are specifically aimed at using Caliper to performance test Hyperledger Fabric networks.
+
+<style>
+.clickable-rectangle {
+    display: block;
+    width: 100%;
+    background-color: #c8d3d5;
+    color: black !important;
+    padding: 20px;
+    text-decoration: none;
+    font-size: 16px;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    transition: background-color 0.3s;
+}
+
+</style>
+
+<a href="https://hyperledger.github.io/caliper/v0.6.0/fabric-tutorial/tutorials-fabric-existing/" class="clickable-rectangle">

```

Comment:
We still have a button here and it still links to the fabric tutorial published by the github pages. If we have anything here, it should a a link to the fabric tutorial that is in this Docs PR.

Reply:
Ok, well I still don't like it being a button as it doesn't look like a button. I'd rather have a link that wasn't hardcoded to the publish site so that it works as expected when I do a local serve and that it links to the content made available in the local serve. If that isn't possible then we should have some test in the index that talks about tutorials and refer the reader to use the navigation bar on the left to select the tutorial. Does that sound ok ?

Reply:
Ok, I think the best thing do do is to remove the button altogether and have no link whatsoever. Just add text to index.md to refer the reader to select the tutorial from the navigation section on the left.

The navigation selection on the left refers to `Existing Fabric Tutorials`. Can we change this to say `Basic Fabric Tutorial` which also appears as the Title for the Fabric Tutorial